src/ray/rpc/worker/core_worker_client_pool.h

```diff
@@ -0,0 +1,76 @@
+// Copyright 2020 The Ray Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#pragma once
+
+#include "absl/base/thread_annotations.h"
+#include "absl/container/flat_hash_map.h"
+#include "absl/synchronization/mutex.h"
+#include "ray/common/id.h"
+#include "ray/rpc/worker/core_worker_client.h"
+
+using absl::optional;
+using std::shared_ptr;
+
+namespace ray {
+namespace rpc {
+
+class CoreWorkerClientPool {
+ public:
+  CoreWorkerClientPool() = delete;
+
+  // Creates a CoreWorkerClientPool based on the low-level ClientCallManager.
```

Comment:
```suggestion
  /// Creates a CoreWorkerClientPool based on the low-level ClientCallManager.
```

Please use the three slash syntax for comments as seen in other header files, here and elsewhere.